scylla-cdc-base/src/test/java/com/scylladb/cdc/model/worker/MockRawChange.java

```diff
@@ -0,0 +1,99 @@
+package com.scylladb.cdc.model.worker;
+
+import com.scylladb.cdc.model.worker.cql.Cell;
+import com.scylladb.cdc.model.worker.cql.Field;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+public class MockRawChange implements RawChange, Comparable<MockRawChange> {
+    private final ChangeSchema changeSchema;
+    private final Map<String, Object> columnValues;
+
+    public MockRawChange(ChangeSchema changeSchema, Map<String, Object> columnValues) {
+        this.changeSchema = changeSchema;
+        this.columnValues = columnValues;
+    }
+
+    @Override
+    public ChangeSchema getSchema() {
+        return changeSchema;
+    }
+
+    @Override
+    public Object getAsObject(ChangeSchema.ColumnDefinition c) {
+        return columnValues.get(c.getColumnName());
+    }
+
+    @Override
+    public Cell getCell(ChangeSchema.ColumnDefinition c) {
+        return new Cell() {
+            @Override
+            public ChangeSchema.ColumnDefinition getColumnDefinition() {
+                return c;
+            }
+
+            @Override
+            public Set<Field> getDeletedElements() {
+                throw new UnsupportedOperationException("Not implemented yet");
+            }
+
+            @Override
+            public boolean hasDeletedElements() {
+                throw new UnsupportedOperationException("Not implemented yet");
+            }
+
+            @Override
+            public boolean isDeleted() {
+                throw new UnsupportedOperationException("Not implemented yet");
+            }
+
+            @Override
+            public ByteBuffer getUnsafeBytes() {
+                throw new UnsupportedOperationException("Not implemented yet");
```

Comment:
Here, realistically, I won't be able to provide any implementation.

Fortunately, no one uses `getUnsafeBytes()` in the tested code.